src/main/java/bio/terra/cda/app/util/NestedColumn.java

```diff
@@ -0,0 +1,64 @@
+package bio.terra.cda.app.util;
+
+public class NestedColumn {
+  private String column;
+  private String unnestClause;
+
+  public NestedColumn() {}
+
+  public NestedColumn(String column, String unnestClause) {
+    this.column = column;
+    this.unnestClause = unnestClause;
+  }
+
+  public static NestedColumn generate(String qualifiedColumnName) {
+    return Builder.generate(qualifiedColumnName);
+  }
+
+  public static class Builder {
+    /**
```

Comment:
This isn't a javadoc comment, so it shouldn't start with `/**`. You can use `//` or `/* ... */` forms for this.

Reply:
Changed to multiline comment